relation_engine_server/utils/wait_for.py

```diff
@@ -0,0 +1,89 @@
+"""
+Block until all dependent services come online.
+"""
+import requests
+import time
+import sys
+from .config import get_config
+
+_CONF = get_config()
+
+
+def wait_for_service(service_list):
+    '''wait for a service or list of services to start up'''
+    timeout = int(time.time()) + 60
+
+    service_conf_list = [get_service_conf(s) for s in service_list]
+
+    while True:
+        try:
+            for service in service_conf_list:
+                name = service['name']
+                url = service['url']
+                if service['auth'] is not None:
+                    requests.get(service['url'], auth=service['auth']).raise_for_status()
+                else:
+                    requests.get(service['url'])
```

Comment:
This line doesn't have the `raise_for_status()` call like line 24

Reply:
That was how it was in the original version -- I don't know it there was a particular reason for that or not. If you aren't aware of there being some reason to not use `raise_for_status`, I'll put it in.

Reply:
I don't remember a particular reason... maybe I was relying on it raising a ConnectionError. But it makes sense to me to use an endpoint where we expect a 2xx response and use raise_for_status

Reply:
Turns out that if you do `raise_for_status`, the tests never proceed as the `auth` and `workspace` servers return 500.